### DISCUSSION TITLE
translating ui and menus

### DISCUSSION THREAD

**Discussion by mnouroozy** [hyperion-project/hyperion.ng#1577] [Q&A] [answered]
hi there 
i wanna translate web ui to persian 
can anybody tell me on which files should i work ?
thanks

**Comment by tpmodding** [comment]
Hey,

you can do this here
https://poeditor.com/join/project/Y4F6vHRFjA
Tasakur dadash

**Reply by mnouroozy** [reply]
> Hey,
> 
> you can do this here https://poeditor.com/join/project/Y4F6vHRFjA Tasakur dadash

thanks dadash 
is there a way to edit files directly?
because i want a smooth ui not just translating words

**Reply by tpmodding** [reply, marked as answer]
what do you mean? our language files are synced with poeditor, so all strings in englisch and you can translate them there, afterwards we sync them back to github and the files are directly
